package.json

```diff
@@ -20,6 +20,12 @@
     ]
   },
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@iconify/icons-mdi": "^1.2.47",
+    "@iconify/icons-uil": "^1.2.3",
+    "@mui/icons-material": "^5.14.6",
+    "@mui/material": "^5.14.6",
```

Comment:
Remove these

Reply:
@Lucif3r-in done

Reply:
> @Lucif3r-in after removing all the checks are failed

Reply:
@Lucif3r-in Deleting those dependencies created problem in code , that's why I am not deleting these 
i have also done npm uninstall  "@mui/icons-material": "^5.14.6",
    npm "@mui/material": "^5.14.6"
    but throws error